lms/templates/google_tag_manager.html

```diff
@@ -0,0 +1,4 @@
+<%doc>
+    Yet, installing google tag manager for microsite(s).
```

Comment:
Can I suggest a different comment here: This file is intentionally left blank. However it could be overridden by a microsite or other theming platform, so please do not remove and corresponding include statement in main.html.